parsers/src/main/scala/uk/gov/ons/addressIndex/crfscala/CrfParser.scala

```diff
@@ -9,12 +9,23 @@ trait CrfParser {
   val crfScala: CrfScalaJniImpl = new CrfScalaJniImpl()
 
   //TODO scaladoc
-  def tag(i: Input, features: CrfFeatures, tokenable: CrfTokenable): String = {
+  def tag(input: Input, features: CrfFeatures, tokenable: CrfTokenable): Seq[CrfTokenResult] = {
+    //todo optimise file
     val currentDirectory = new java.io.File(".").getCanonicalPath
     val modelPath = s"$currentDirectory/parsers/src/main/resources/addressCRFA.crfsuite"
-    val actual = parse(i, features, tokenable)
+    val actual = parse(input, features, tokenable)
     val augmentedActual = augmentCrfJniInput(actual)
```

Comment:
Yes, right, but it could be named also as `val a =` and still be used as an input in the next line.
What I mean is that I don't understand the meaning of "augmented" in this context and would like it to be improved. If the meaning is obvious (and the change is not needed), I'd like to know it.